scripts/generate-docs.sh

```diff
@@ -63,49 +65,38 @@ rm docs/mkdocs/index.md || true
 # deprecated in 3.1.0 but mkdocstring depends on this method
 
 if [ -z "$SKIP_INSTALL" ]; then
-  # TODO: reimplement this installation sequence so
-  # we have an updated list of pip requirements
-
-  # zenml integration install -y feast
-  # zenml integration install -y label_studio
-  # zenml integration install -y bentoml
-  # zenml integration install -y airflow
-  # zenml integration install -y kserve
-  # zenml integration install -y --ignore-integration feast --ignore-integration label_studio --ignore-integration kserve --ignore-integration airflow --ignore-integration bentoml
-  # pip install jinja2==3.0.3 protobuf==3.20.0 numpy~=1.21.5
-  # pip install typing-extensions --upgrade
-  # pip install feast --upgrade  # The integration feast version has unsupported googleapis-common-protos >=1.52.* requirement
-  
-  # TEMPORARY FIX
-  pip install -e ".[server,dev,secrets-aws,secrets-gcp,secrets-azure,secrets-hashicorp,s3fs,gcsfs,adlfs]"
-  pip install -r docs/requirements-docs-frozen.txt
+  pip3 install -e ".[server,dev]"
+  pip3 install "Jinja2==3.0.3"
```

Comment:
is it so precarious that we have to pin it so specifically?

Reply:
I can test without it, but failed on local clean env always

Reply:
Yeah, collapses without pinned version: `error: cannot import name 'contextfunction' from 'jinja2' (/opt/hostedtoolcache/Python/3.11.7/x64/lib/python3.11/site-packages/jinja2/__init__.py)` same as on my local